util/format/format.go

```diff
@@ -178,12 +178,10 @@ func OutputFormat(s string) string {
 	replace := map[string]string{
```

Comment:
Move this variable out, we don't need create a map every time `OutputFormat` is called.